.github/workflows/ledger.yml

```diff
@@ -1,11 +1,11 @@
 name: CI
 
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
+  # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [ master ]
```

Comment:
Hello @carlosala, you asked us to rename the default branch to main before merging but that does not work well with our methodology.
We take external contributions on a develop branch, then from that branch deploy the app on a testing provider to validate it.
Then, only when that app is moved to the release provider, the develop branch is then merged to master. (Which I renamed main in this case).
So you should add the triggers also on develop.

Reply:
It should be solve now 😄